Upstatement/skela-wp-theme
# Find and replace script

## Changes

https://github.com/Upstatement/skela-wp-theme/issues/35

Added a script to automate the case-sensitive search and replace required when setting up a new Skela project!

## How To Test

- Run `./bin/rename-theme` and enter your hypothetical new theme name
- Do a global search of `skela` in your repo and make sure the only instances are ones we intended to keep (i.e. the `img` in `front-page.twig`)
- Make sure `./bin/install` succeeds
